assets.md

```diff
@@ -118,3 +118,11 @@ Link: https://project.lfcla.com/#/project/a0941000002wBz4AAE/cla
 ### Docker Hub
 
 - We publish images from CI to Docker hub using https://hub.docker.com/u/otelbot account. The bot is registered using cncf-opentelemetry-tc@lists.cncf.io email address and Technical Committee members are owners of this account. The Admin for bot security credentials for CI is @tigrannajaryan
+
+### OpenTelemetry Bot
+
+This is a bot that has signed the CNCF CLA and can be used as the author of automated PRs.
```

Comment:
Could you mention which repos are making use of this account?
This would make it easier to adapt permissions or rotate tokens if we know where things might break and need to be adapted.

Reply:
So far:
* opentelemetry-java
* opentelemetry-java-instrumentation
* opentelemetry-java-contrib
* opentelemetry-python
* opentelemetry-sandbox-web-js
* opentelemetry.io (soon)

What do you think of going ahead and making it an org secret? (e.g. `BOT_TOKEN`, `OPENTELEMETRYBOT_TOKEN`, ...)

Reply:
Thank you for the list!

It would be an option. On the other hand, if they have to be added manually, there is better control over where they are being used in case something needs to be changed (although grepping for the usage of `OPENTELEMETRYBOT_TOKEN` should be just as fine). Also, the risk of the credentials being misused is restricted to only the repos where it was manually added. If it's a repo secret, individual tokens can be assigned to each repo, each with their own permissions (and maybe separate audit logs, not sure about that). Since new repos won't be set up to use the bot that often, I think the maintenance effort is small enough to keep the manual process and have the other benefits instead.
We could discuss that further on the maintainer's call or a dedicated issue if you'd like.

Reply:
I added a note clarifying that @opentelemetrybot does not need to be (and should not be) given any permissions to any OpenTelemetry repositories. If the PAT leaked, the worst that could be done is to submit a PR as @opentelemetrybot, which would still require a maintainer to approve and merge it.

If you think this should have an individual token per repo, I'll send out new keys to folks, and create a list of which repo is using which PAT in the GC-owned 1password account (did you want the list of just the repos to be maintained in this doc also?).

If you don't think it needs an individual token per repo, I think we can still restrict an org secret to specific repos if we want to track which ones are using it.

Reply:
Good point about the permissions! f we can be that restrictive about them, then I think an org secret should work and would also require lower maintenance. I was not aware of the option to restrict them to select repositories but if we do it like that, I think we should even be fine without having to manually (=error-prone) maintain such a list in 1Password or here.

Reply:
@arminru I made some updates to incorporate the org secret approach, ptal, thx

Reply:
LGTM, thank you!